tests/test_dispersion.py

```diff
@@ -1,267 +1,46 @@
-        "mass": (16.241, 0.1),
-        "inertiaI": (6.6, 0.66),
-        "inertiaZ": (0.0351, 0.00351),
-        "radius": (0.0635, 0.001),
-        "powerOffDrag": ["data/calisto/powerOffDragCurve.csv"],
-        "powerOnDrag": ["data/calisto/powerOnDragCurve.csv"],
-        "powerOffDragFactor": (1, 0.033),
-        "powerOnDragFactor": (1, 0.033),
-        "centerOfDryMassPosition": (0, 0.001),
-        "motor_position": (-1.255, 0.001),
-        "nose_name_kind": ["vonKarman"],
-        "nose_name_length": (0.55829, 0.001),
-        "nose_name_position": (1.278, 0.001),
-        "finSet_name_n": [4],
-        "finSet_name_rootChord": (0.12, 0.001),
-        "finSet_name_tipChord": (0.04, 0.001),
-        "finSet_name_span": (0.1, 0.001),
-        "finSet_name_position": (-1.04956, 0.001),
-        "finSet_name_airfoil": [None],
-        "tail_name_topRadius": (0.0635, 0.001),
-        "tail_name_bottomRadius": (0.0435, 0.001),
-        "tail_name_length": (0.06, 0.001),
-        "tail_name_position": (-1.194656, 0.001),
-        "parachute_Main_CdS": (10, 2),
-        "parachute_Main_trigger": [mainTrigger],
-        "parachute_Main_samplingRate": (105, 0),
-        "parachute_Main_lag": (1.5, 0),
-        "parachute_Main_noise": [(0, 8.3, 0.5)],
-        "parachute_Drogue_CdS": (1, 0.3),
-        "parachute_Drogue_trigger": [drogueTrigger],
-        "parachute_Drogue_samplingRate": (105, 0),
-        "parachute_Drogue_lag": (1.5, 0),
-        "parachute_Drogue_noise": [(0, 8.3, 0.5)],
-        "positionFirstRailButton": (0.2, 0.001),
-        "positionSecondRailButton": (-0.5, 0.001),
-        "railButtonAngularPosition": (45, 0),
-        "inclination": (85, 1),
-        "heading": (90, 2)
-        # "terminateOnApogee":,
-        # "maxTime":,
-        # "maxTimeStep":,
-        # "minTimeStep":,
-        # "rtol":,
-        # "atol":,
-        # "timeOvershoot":,
-        # "verbose":,
-    }
+    Returns
+    -------
+    None
+    """
 
-    test_disp.run_dispersion(
+    dispersion.run_dispersion(
         number_of_simulations=10,
-        dispersion_dictionary=disp_dict,
+        append=False,
     )
 
-    test_disp.import_results()
-
-    assert test_disp.print_results() == None
-    assert test_disp.allInfo() == None
+    assert dispersion.allInfo() == None
+    assert (
+        dispersion.exportEllipsesToKML(
+            filename="test_dispersion_class.kml", origin_lat=0, origin_lon=0
+        )
+        == None
+    )
+    assert dispersion.errors_log == []
+    assert len(dispersion.outputs_log) == 10
+    assert dispersion.plotEllipses(save=True) == None
 
-    os.remove("test_obj_defined.disp_inputs.txt")
-    os.remove("test_obj_defined.disp_outputs.txt")
-    os.remove("test_obj_defined.disp_errors.txt")
-    os.remove("test_obj_defined.svg")
+    # Delete the test file
+    # TODO: can't delete the files because they were not closed, bad practice!
+    # os.remove("test_dispersion_class.disp_errors.txt")
```

Comment:
Hey, it seems these files have never been closed in the Dispersion class

Reply:
@MateusStano I noticed that you used a lot of `blablabla = open('flename', ...)`, this is actually not the best approach possible ([see](https://realpython.com/why-close-file-python/)) since it doesn't guarantee the files to be closed.
Instead, I think one may use `with open(...)`, and then passing the content of the file to a string, which could be added by simply using the `+`  operand.

Let m know what you think about it

Reply:
Fixed in https://github.com/RocketPy-Team/RocketPy/pull/336/commits/4e8319e6bbaa1ef7afef23386fe4b5076ba9c78f